File: admin/src/main/java/com/weicoder/admin/po/LogsLogin.scala
package com.weicoder.admin.po

import javax.persistence.Entity
import org.hibernate.annotations.DynamicInsert
import org.hibernate.annotations.DynamicUpdate
import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component
import org.springframework.beans.factory.config.ConfigurableBeanFactory
import com.weicoder.base.entity.EntityIp
import com.weicoder.base.entity.EntityShards
import com.weicoder.base.entity.EntityUserId
import com.weicoder.site.entity.base.BaseEntityIdTime
import scala.beans.BeanProperty

/**
 * 登录日志实体
 * @author WD
 * @since JDK7
 * @version 1._ 2_11-_4-_3
 */
@Component
@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
@Entity
@DynamicInsert
@DynamicUpdate
class LogsLogin extends BaseEntityIdTime with EntityIp {
  // 登录IP
  @BeanProperty
  var ip: String = _
  // 用户ID
  @BeanProperty
  var userId: Integer = _
  // 状态
  @BeanProperty
  var state: Integer = _
  // 名称
  @BeanProperty
  var name: String = _
}

File: admin/src/main/java/com/weicoder/admin/po/LogsLogin.java
package com.weicoder.admin.po;

import javax.persistence.Entity;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;

import org.hibernate.annotations.DynamicInsert;
import org.hibernate.annotations.DynamicUpdate;
import org.springframework.beans.factory.config.ConfigurableBeanFactory;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

import com.weicoder.frame.entity.EntityIp;
import com.weicoder.frame.entity.EntityUserId;
import com.weicoder.frame.entity.base.BaseEntityIdTime;

/**
 * 登录日志实体
 * @author WD
 * @since JDK7
 * @version 1.0 2011-04-03
 */
@Component
@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
@Entity
@DynamicInsert
@DynamicUpdate
@Getter
@Setter
public class LogsLogin extends BaseEntityIdTime implements EntityIp, EntityUserId {
	// 登录IP
	private String	ip;
	// 用户ID
	private Integer	userId;
	// 状态
	private Integer	state;
	// 名称
	private String	name;
}
